src/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const BASE_URL = 'https://project-news-liliia.onrender.com/api';
+
+export const fetchArticles = async () => {
+  try {
+    const response = await axios.get(`${BASE_URL}/articles`);
+    return response.data.articles;
+  } catch (error) {
+    console.error('Error fetching articles:', error);
+    throw error;
+  }
+};
```

Comment:
I would recommend looking into the following notes on a nicer way to do this:

l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=creating-an-axios-instance